src/uu/du/src/du.rs

```diff
@@ -361,126 +384,187 @@ fn convert_size_other(size: u64, _multiplier: u64, block_size: u64) -> String {
     format!("{}", ((size as f64) / (block_size as f64)).ceil())
 }
 
+fn get_usage() -> String {
+    format!(
+        "[OPTION]... [FILE]...
```

Comment:
there are brackets missing, the output should be
```
du [OPTION]... [FILE]...
du [OPTION]... --files0-from=F
```

Reply:
Good catch!